MetadataProcessor.Shared/Tables/nanoAssemblyReferenceTable.cs

```diff
@@ -17,6 +17,13 @@ namespace nanoFramework.Tools.MetadataProcessor
     public sealed class nanoAssemblyReferenceTable :
         nanoReferenceTableBase<AssemblyNameReference>
     {
+        //////////////////////////////////////////////////////////////////////////////////////////////
+        //////////////////////////////////////////////////////////////////////////////////////////////
+        // when updating this size here need to update matching define in nanoCLR_Types.h in native //
+        private const int sizeOf_CLR_RECORD_ASSEMBLYREF = 10;
```

Comment:
I've seen few times comments like this (even a full file with classes).
Maybe for maintainability we can add a specific tag like "TO_UPDATE_NATIVE" in the comments so when searching for tag over the code, we can find it easily (and document this tag in the main readme).
Because if we do a table or anything like this as a summary, we'll never update. So, there are more chances that a tag like this will not be forgotten too much :-D

Reply:
Good point. Will do.